services/grafana/auth_server.go

```diff
@@ -70,7 +70,7 @@ var rules = map[string]role{
 	"/graph":           none,
 	"/qan":             none,
 	"/swagger":         none,
-
+	"/logs.zip":        admin,
 	// "/auth_request" and "/setup" have auth_request disabled in nginx config
```

Comment:
That group was intended to be the same as mustSetupRules.

Please add a comment on top of that, and keep a blank line before logs.zip.

Please also add "/victoriametrics" to mustSetupRules

Reply:
+1

Reply:
@AlekSi I couldn't find `/victoriametrics` in nginx

Reply:
https://github.com/percona/pmm-server/blob/f09e39bb7de10ca69174a9ad96445953aacf512f/nginx.conf#L122-L126